src/tutorials/volume-snapshots-how-to-create-and-use-data-clones-in-okteto.mdx

```diff
@@ -0,0 +1,311 @@
+
+
+## Step 2: Populate the Application Database
+
+Before taking a snapshot, add sample data. This is the data that we'll be including in the Development Environments that we'll create in the next steps.
+
+```console
+curl -X 'POST' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "Donuts",
+  "ingredients": ["Flour", "Milk", "Sugar", "Vegetable Oil"]
+}'
+```
+
+```console
+curl -X 'POST' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json' \
+  -H 'Content-Type: application/json' \
+  -d '{
+  "name": "Pancake",
+  "ingredients": ["Flour", "Milk", "Sugar", "Baking Powder", "Vegetable Oil"]
+}'
+```
+
+Verify that the database has been populated by retrieving the list of recipes present:
+
+```console
+ curl -X 'GET' \
+  'https://fastapi-snapshot-demo-yournamespace.YOURSUBDOMAIN/recipe' \
+  -H 'accept: application/json'
+```
+
+Expected response:
+
+```json
+{
+  "data": [
+    {
+      "id": "60ce68651eeddf1c5ab796d1",
+      "name": "Donuts",
+      "ingredients": [
+        "Flour",
+        "Milk",
+        "Sugar",
+        "Vegetable Oil"
+      ]
+    },
+    {
+      "id": "60ce689c1eeddf1c5ab796d2",
+      "name": "Pancake",
+      "ingredients": [
+        "Flour",
+        "Milk",
+        "Sugar",
+        "Baking Powder",
+        "Vegetable Oil"
+      ]
+    }
+  ]
+}
+```
+
+## Step 3: Create a Database Snapshot
+
+A Volume Snapshot will capture the database state at this moment.
+
+Create a new manifest file `snapshot.yml` in the `manifests` folder and add the following to it:
+
+```yaml
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: mongo-snapshot
+spec:
+  volumeSnapshotClassName: okteto-snapshot-class
+  source:
+    persistentVolumeClaimName: data-mongodb-0
+```
+
+In the Volume Snapshot Manifest above, you are creating a Volume Snapshot named `mongo-snapshot`, the snapshot will be created from the persistent volume `data-mongodb-0` which was created by the MongoDB manifest.
+
+
+Next, apply the Volume Snapshot from your terminal:
+
+```console
+kubectl apply -f manifests/snapshot.yml
+```
+
+Finally, verify that the volume snapshot has been created successfully.
+
+```console
+kubectl get volumesnapshot
+```
+
+✅ Now we have a snapshot that can be used for new development environments!
+
+
+## Step 4: Deploy a Development Environment Using the Snapshot
+
+### 1. Create a New Namespace for Development
+
+```console
+okteto namespace create data-clone-$YOUR_OKTETO_USERNAME
+```
+
+### 2. Modify the Application Manifest to Use the Snapshot
+
+Create a new folder in the parent directory and copy `k8s.yml` and `mongo.yml` into it:
+
+```console
+mkdir data-clone-manifest
+cp manifests/{k8s.yml,mongo.yml} data-clone-manifest
```

Comment:
what if we include the manifests already on a separate folder ? or do you think there's value on the user manually modifying it as a learning experience?